### PR TITLE
Special packages

### PR DESCRIPTION
**Pull request by nvwls**
Support gpg-pubkey "RPMs" which are the result of `rpm --import KEY`

Change-Id: I1c64bb2ce111bc1b81dafc32dbc0457eee309b68
Signed-off-by: joe.nuspl <joe.nuspl@workday.com>

### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [x] New functionality includes tests
- [x] All tests pass
- [x] RELEASE\_NOTES.md, has been updated if required (not required for bugfixes, required for API changes)
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+inherit_gem:
+  chefstyle: config/chefstyle.yml
```
Comment (tas50) [inline .rubocop.yml]: Are you trying to run rubocop here on the cli and still get chefstyle?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -35,3 +35,9 @@ The EC2 plugin now fetches the AWS Account ID in addition to previous instance m
 ## GCC Detection
 
 GCC detection has been improved to collect additional information, and to not prompt for the installation of Xcode on macOS systems
+
+## Support multiple kernel RPMs
+The last kernel installed will be accessible via `[:packages]['kernel']`. Additional kernels can be accessed via `[:packages]['kernel-VERSION-RELEASE']`.
```
Comment (tas50) [inline RELEASE_NOTES.md]: Can you stick this all back at the top of the released notes please

File: .rubocop.yml (hunk 1 above)
Comment (nvwls) [reply .rubocop.yml]: yes

**Comment by tas50** [comment]
@nvwls We merged a similar fix that allows multiple versions of the same package name. That'll ship in 15.2. I'm going to close this one out now.

**Comment by glensc** [comment]
link https://github.com/chef/ohai/pull/1369

**Pull request closed without merging**